system/CodeIgniter.php

```diff
@@ -830,7 +830,7 @@ protected function startController()
         $this->benchmark->start('controller_constructor');
 
         // Is it routed to a Closure?
-        if (is_object($this->controller) && (get_class($this->controller) === 'Closure')) {
+        if (is_object($this->controller) && (get_class($this->controller) === Closure::class)) {
```

Comment:
There isn't by chance a way to "exempt core classes without namespaces" is there?

Reply:
Not yet, it possibly need to have configured, eg skip_native_class

Reply:
It actually configurable, but currently need to manually define what need to be skipped https://github.com/rectorphp/rector-src/blob/de3c9b6f90ccc901c135a7d9786c4a9ea772bd4d/rules/Php55/Rector/String_/StringClassNameToClassConstantRector.php#L118

Reply:
@MGatner I updated to exclude specific native classes https://github.com/codeigniter4/CodeIgniter4/pull/5813/commits/0176bb4224c3fcdf86b5c526808a1d754035154b